geniesp/bpc_redcap_export_mapping.py

```diff
@@ -1447,7 +1448,9 @@ def run(self):
 
         self.create_gene_panels(subset_sampledf["SEQ_ASSAY_ID"].unique())
         # Make sure to re download all the metadata files again
-        self.download_metadata_files()
+        # TODO: need to add folder path to write meta files to...
+        metafiles.create_cbio_metafiles(study_identifier="insert_study_id?")
```

Comment:
@hhunterzinck Couple of things that we still need to work out

1. I'm not sure what I created is truly the best way to go about writing this code.  Please take a look and let me know what you think.  I wonder if it would be helpful to have a function that creates a metadata file per filename
2. the study id is custom per study, this needs to be added to the class
3. Still need to add a parameter to write the metadata files to the right location.